parse_date.re

```diff
@@ -2400,6 +2397,19 @@ timelib_time *timelib_parse_from_format_with_map(char *format, char *string, siz
 		}
 	}
 
+	/* Handle day of year */
+	if ( day_of_year != TIMELIB_UNSET ) {
+		/* Check for mixing day-of-year with month or day-of-month */
+		if (s->time->m != TIMELIB_UNSET || s->time->d != TIMELIB_UNSET) {
+			add_pbf_error(s, TIMELIB_ERR_MIX_DAY_OF_YEAR_WITH_M_D, "Mixing of day-of-year with month or day is not allowed", string, ptr);
+		}
+		else {
```

Comment:
`} else {` goes on one line. I would also prefer that the part in the `if` contains the success case, and the part in the `else` the failure case.